### PR TITLE
Update conv3d API

### PR DESCRIPTION
**Pull request by mls1999725**
### PR types

Bug fixes
### PR changes

APIs
### Describe

Delete keyword of fluid.

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -1751,11 +1751,10 @@ def conv3d(input,
     Examples:
         .. code-block:: python
 
-          import paddle.fluid as fluid
           import paddle
           paddle.enable_static()
-          data = fluid.data(name='data', shape=[None, 3, 12, 32, 32], dtype='float32')
-          conv3d = fluid.layers.conv3d(input=data, num_filters=2, filter_size=3, act="relu")
+          data = paddle.static.data(name='data', shape=[None, 3, 12, 32, 32], dtype='float32')
+          conv3d = paddle.static.nn.conv3d(input=data, num_filters=2, filter_size=3, act="relu")
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py]: 补上 print(conv3d.shape) # (-1, 2, 10, 30, 30) 这一句吧

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -4210,11 +4209,10 @@ def conv3d_transpose(input,
     Examples:
        .. code-block:: python
 
-          import paddle.fluid as fluid
           import paddle
           paddle.enable_static()
-          data = fluid.data(name='data', shape=[None, 3, 12, 32, 32], dtype='float32')
-          conv3d_transpose = fluid.layers.conv3d_transpose(input=data, num_filters=2, filter_size=3)
+          data = paddle.static.data(name='data', shape=[None, 3, 12, 32, 32], dtype='float32')
+          conv3d_transpose = paddle.static.nn.conv3d_transpose(input=data, num_filters=2, filter_size=3)
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py]: 同上

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
LGTM for fluid.layers.conv3d without 'core.ops' because it is used in static graph.

**Pull request merged**